### PR TITLE
Merges with last slate version

### PR DESCRIPTION
**Pull request by waldemarnt**
I just update the slate version to ensure that the documentation is up to date.
Some points were fixed, as:
- [x] Responsive images
- [x] Table colors
- [x]  Right panel

Header
Before
screen shot 2016-08-18 at 2 47 19 pm

After:
screen shot 2016-08-18 at 2 47 23 pm

Tables

Before:
screen shot 2016-08-18 at 2 50 14 pm

After:
screen shot 2016-08-18 at 2 50 19 pm

### CODE REVIEW

**Comment by glauco** [comment]
LGTM! :)

**Comment by lucaslago** [comment]
👍

File: Gemfile [hunk 1]
```diff
@@ -2,12 +2,11 @@ source 'https://rubygems.org'
 
 # Middleman
 gem 'rake', '~> 11.1.2'
-gem 'middleman', '~> 4.0.0'
+gem 'middleman', '~>4.0.0'
 gem 'middleman-gh-pages', '~> 0.0.3'
 gem 'middleman-syntax', '~> 2.1.0'
 gem 'middleman-autoprefixer', '~> 2.7.0'
-gem 'middleman-sprockets', '~> 4.0.0.rc'
+gem "middleman-sprockets", "~> 4.0.0.rc"
```
Comment (lucaslago) [inline Gemfile]: Maybe we could use single quotes here to maintain the pattern, what do you think?

File: LICENSE [hunk 2]
```diff
@@ -1,23 +1,3 @@
-Copyright (c) 2016 AGCO Corporation
```
Comment (lucaslago) [inline LICENSE]: I believe this was a merging error. Is this deletion correct?

File: Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,39 @@
+Vagrant.configure(2) do |config|
```
Comment (lucaslago) [inline Vagrantfile]: Do we need this Vagrantfile?

File: source/includes/_errors.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Errors
+
+<aside class="notice">This error section is stored in a separate file in `includes/_errors.md`. Slate allows you to optionally separate out your docs into many files...just save them to the `includes` folder and add them to the top of your `index.md`'s frontmatter. Files are included in the order listed.</aside>
```
Comment (lucaslago) [inline source/includes/_errors.md]: This seems like an example markdown. Can we delete it?

File: LICENSE (hunk 2 above)
Comment (jeffhsta) [inline LICENSE]: Is that right, remove AGCO's license?

File: Vagrantfile (hunk 3 above)
Comment (jeffhsta) [inline Vagrantfile]: We are not using Vagrant, do you think that adds it make sense?

File: source/index.html.md [hunk 5]
```diff
@@ -35,6 +35,7 @@ includes:
 search: true
 ---
 
+
```
Comment (jeffhsta) [inline source/index.html.md]: There is a double blank like here

File: LICENSE (hunk 2 above)
Comment (waldemarnt) [inline LICENSE]: @lucaslago @jeffhsta thanks, probably was removed when merged

**Comment by lucaslago** [comment]
👍

**Pull request merged**